modules/compute/vm-instance/main.tf

```diff
@@ -133,6 +133,24 @@ resource "google_compute_instance" "compute_vm" {
     nic_type   = local.enable_gvnic ? "GVNIC" : null
   }
 
+  dynamic "network_interface" {
+    for_each = var.additional_network_interfaces
+
+    content {
+      network            = lookup(network_interface.value, "network", null)
+      subnetwork         = lookup(network_interface.value, "subnetwork", null)
+      subnetwork_project = lookup(network_interface.value, "subnetwork_project", null)
+      network_ip         = lookup(network_interface.value, "network_ip", null)
+      nic_type           = lookup(network_interface.value, "nic_type", null)
+      stack_type         = lookup(network_interface.value, "stack_type", null)
+      queue_count        = lookup(network_interface.value, "queue_count", null)
+      dynamic "access_config" {
```

Comment:
I would rather have an external variable, disable_public_ips that determine default values here if the interface is not defined or if the value vas omitted, but I would not rename the structure member. Furthermore, if a user passes access config, it would be ignored here. 
Access config contains other interesting options such as an IP address, a networking tier, or the domain name.

Reply:
access_config has been added, and in fact it now supports multiple if desired. 

I chose to only apply `var.disable_public_ips` (top level variable) to the network interface as defined by `subnetwork_self_link` and `network_self_link`. If a value is passed into `network_interfaces`, the assumption is that the user knows what they are doing  and wouldn't want their configuration overwritten. What do you think about that?

Reply:
agree, as long as it is well documented in the variables description.